Automattic/wp-calypso
# Remove white glove references in code

#### Changes proposed in this Pull Request

* The "Expert guidance" test was reverted in https://github.com/Automattic/wp-calypso/pull/44022, but there's one more white glove reference which stuck on. This PR removes it.

#### Testing instructions



* Go through signup and ensure that the existing post signup upsells - Quick Start upsell and Business bump upsell - work well. Purchase a Personal plan to see the Quick Start upsell, and purchase a Premium plan to see the Business bump upsell.
